be/src/exec/workgroup/work_group.h

```diff
@@ -244,6 +252,14 @@ class WorkGroupManager {
 
     std::unique_ptr<WorkerOwnerManager> _driver_worker_owner_manager;
     std::unique_ptr<WorkerOwnerManager> _scan_worker_owner_manager;
+
+    bool _init_metrics = false;
```

Comment:
Maybe the `std::once_flag` is more precise to describe the usage and concurrent-safe ?

Reply:
OK